widgetastic_patternfly.py

```diff
@@ -1257,3 +1258,33 @@ def item_select(self, item, handle_alert=None):
 
     def __repr__(self):
         return '{}({!r})'.format(type(self).__name__, self.text)
+
+
+class BootstrapSwitch(Checkbox):
+    """ represents checkbox like switch control.
+    widgetastic checkbox doesn't work right for this control.
+    So, this widget is some kind of enhancement
+    .. code-block:: python
+
+        switch = BootstrapSwitch(id="default_tls_verify"')
+        switch.fill(True)
+        switch.read()
+    """
+    ROOT = ParametrizedLocator('//div[contains(@class, "bootstrap-switch-container") and '
+                               '{@input}]')
+
+    def __init__(self, parent, id=None, name=None, logger=None):
+        if not (id or name):
```

Comment:
i'm under the impression that id/name based selection is getting more common, we might want a abstraction for that if similar implementations pop up